docs/source/participate.md

```diff
@@ -0,0 +1,43 @@
+## Tutorial to participate in a challenge in EVALAI
```

Comment:
@gautamjajoo I have added the review in PR #1359. Please remove that question from there and incorporate those changes in this PR. Also, please let me know if you face any issues.

Reply:
Ok..